Makefile

```diff
@@ -78,6 +78,17 @@ else
 endif
 	python setup.py install
 
+q2dtor:
+	@ echo "\nInstalling Q2DTor...\n"
+	@ echo "Q2DTor is a software for calculating the partition functions and themodynamic properties\
+	of molecular systems with two or more torsional modes developed by David Ferro Costas (david.ferro@usc.es)\
+	 and Antonio Fernandez Ramos (qf.ramos@usc.es) at the Universidade de Santiago de Compostela. Arkane can\
+	  integrate Q2DTor to compute the quantum mechanical partition function of 2D rotors.  \n\nFor use of Q2DTor\
+		 and HinderedRotor2D within Arkane please cite:  \n\nD. Ferro-Costas, M. N. D. S.Cordeiro, D. G. Truhlar, A.\
```

Comment:
Does it ignore all the leading spaces? If so, do we need them?

Reply:
i made a small fix, but other than line before the citation everything looks normal.